DoublyLinkedList/header.h

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
+using namespace std;
+// Node Creation
+class Node{                                                     //class
```

Comment:
This should be renamed to something better. as with previous comment, imagine someone using your library to create a DLL.. they will be calling Node to create a DLL. assuming you call other data structures (queue etc) as Node as well, they will find it difficult to use your library.

Reply:
Resolved that issue sir